week1/armstrong.rb

```diff
@@ -9,5 +9,23 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+    puts "Enter a number"
+    n=gets.chomp.to_i
```

Comment:
@diviz11 kindly follow the instructions given for the assignment. The number to be verified is passed as a parameter to the function and you do not need to take in input from the user